javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const width = 5;
+const height = 4;
+const area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+const bill = 10;
+const payment = 15;
+const change = bill - payment;
+console.log('value of change:', change);
+console.log('typeof change', typeof change);
+const quizzes = 6;
+const midterm = 11;
+const final = 7;
+const grade = quizzes + midterm + final;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+const firstName = `Andrew `;
+const lastName = `Ortiz`;
+const fullName = `${firstName}${lastName}`;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+const ph = 10;
+const isAcidic = ph < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+const headCount = 200;
+const isSparta = true;
```

Comment:
Rather than assigning `true` to `isSparta`, you should be assigning it an expression of `headCount === 300`